.env.development

```diff
@@ -1,3 +1,2 @@
 ARCHIVE_URL=https://archive.cnx.org/
 REACT_APP_ARCHIVE_URL=/contents/
-PORT=3000
```

Comment:
you can leave the port in development, its possible to run the pre-renderer in development it'll just render all of college physics instead of just the fixtures